bootstrap/config/config.go

```diff
@@ -216,7 +216,7 @@ func (cp *Processor) Process(
 	// Now load the private config from a local file if any of these conditions are true
 	if !useProvider || !cp.providerHasConfig || cp.overwriteConfig {
 		filePath := GetConfigFileLocation(cp.lc, cp.flags)
-		configMap, err := cp.loadConfigYamlFromFile(filePath, defaultTimeout, secretProvider)
+		configMap, err := cp.loadConfigYamlFromFile(filePath, file.DefaultTimeout, secretProvider)
```

Comment:
We could use the **SERVICE_REQUESTTIMEOUT** env var here (normal env override), if set, otherwise use the default you are using.

Add GetRequestTimeout to environment/variables.go that encapsulates this.